RFC-003-SWAP-basic.md

```diff
@@ -119,6 +119,7 @@ When `comit-rfc-003` is used as the value for `protocol` for a SWAP REQUEST mess
 | `beta_redeem_identity`  | `β::Identity`       | The identity on β that **B** will be transferred to when the β-HTLC is activated with the correct secret  |
 | `secret_hash`           | `hex-encoded-bytes` | The output of calling `hash_function` on the secret                                                       |
 
+Implementations SHOULD consider messages with values of `alpha_expiry` or `beta_expiry` that are in the past to be invalid.
```

Comment:
Note I added this line to RFC003 as part of this PR because I realised we don't recommend any checking of this at all.

Reply:
Woops. I never intended for this change to go into the bitcoin PR but now that it's happened it's not worth the trouble of removing this change.